src/aiometer/_impl/amap.py

```diff
@@ -16,10 +16,15 @@
 from .run_on_each import run_on_each
 from .types import T, U
 
-try:
-    from typing_extensions import Literal  # Python 3.7.
-except ImportError:  # pragma: no cover
-    from typing import Literal  # type: ignore
+if sys.version_info > (3, 8):  # pragma: no cover
```

Comment:
it's better to use this approach than try/catch because mypy and pyupgrade both understand them

Reply:
I've seen this suggested before, sounds legit. :-)